src/NuGetPackageConfigConverter/Services/PackageConfigEntry.cs

```diff
@@ -5,11 +5,11 @@ public struct PackageConfigEntry
         public PackageConfigEntry(string id, string version)
         {
             Id = id;
-            Version = version;
+            Version = version??"";
```

Comment:
typo

Reply:
No, trying to avoid it sending in version as null.

Reply:
got it. Without spaces I didn't realize it was the null coalescing operator

Reply:
lol.  it was a bit tight :-)